{{cookiecutter.project_name}}/environments/kolla/configuration.yml

```diff
@@ -30,9 +30,9 @@ openstack_service_workers: 5
 es_heap_size: "4g"
 
 # neutron
-neutron_plugin_agent: "openvswitch"
-
-enable_neutron_agent_ha: "yes"
```

Comment:
Do i understand it right? This parameter has no effect for ovn?

Reply:
The parameter is only required for Neutron L3 agents.

```
ansible/group_vars/all.yml.orig:enable_neutron_agent_ha: "no"
ansible/group_vars/all.yml:enable_neutron_agent_ha: "no"
ansible/roles/neutron/tasks/precheck.yml:    - enable_neutron_agent_ha | bool
ansible/roles/neutron/templates/neutron-l3-agent-wrapper.sh.j2:{% if enable_neutron_agent_ha | bool %}
ansible/roles/neutron/templates/neutron.conf.j2:{% if enable_neutron_agent_ha | bool %}
ansible/roles/neutron/templates/l3_agent.ini.j2:{% if enable_neutron_agent_ha | bool %}
doc/source/reference/networking/neutron.rst:   enable_neutron_agent_ha: "yes"
etc/kolla/globals.yml.orig:#enable_neutron_agent_ha: "no"
etc/kolla/globals.yml:#enable_neutron_agent_ha: "no"
releasenotes/notes/bp-support-network-ha-1a771d735a268219.yaml:  - New option enable_neutron_agent_ha added to enable/disable dhcp/l3 agent
```